.github/workflows/docker-image.yml

```diff
@@ -22,5 +22,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Build the Docker image
-      run: docker build . --file Dockerfile --tag axistools/caddy-bundle:$(date +%s)
+
+#    - name: Build the Docker image
+#      run: docker build . --file Dockerfile --tag axistools/caddy-bundle:$(date +%s)
```

Comment:
Should you just remove these instead?

Reply:
Sorry, I did not mean for you to start reviewing this just yet. I just wanted to trig the github action without waiting for the entire docker build. Will fix this.